mono/utils/Makefile.am

```diff
@@ -149,6 +149,8 @@ monoutils_sources = \
 	mono-threads-coop.h	\
 	mono-tls.h	\
 	mono-tls.c	\
+	mono-utils-debug.c \
+	mono-utils-debug.h \
```

Comment:
These need to be added to the vcxprojs in msvc/ too. The Windows build is broken because of this.